core/src/main/java/org/springframework/security/provisioning/JdbcUserDetailsManager.java

```diff
@@ -603,4 +621,14 @@ private void validateAuthorities(Collection<? extends GrantedAuthority> authorit
 		}
 	}
 
+	@Override
+	public UserDetails updatePassword(UserDetails user, String newPassword) {
```

Comment:
```suggestion
	@Override
	/**
	  * comments like "updatePassword only when {@link #enableUpdatePassword} is true?
	  */
	public UserDetails updatePassword(UserDetails user, String newPassword) {
```
nit. how about adding comment about `updatePassword ` flag?